src/workers/parser/pausePoints.js

```diff
@@ -50,6 +51,52 @@ function getStartLine(node) {
   return node.loc.start.line;
 }
 
+// Finds the first call item in a step expression so that we can step
+// to the beginning of the list and either step in or over. e.g. [], x(), { }
+function isFirstCall(node, parentNode, grandParentNode) {
+  let children = [];
+  if (t.isArrayExpression(parentNode)) {
+    children = parentNode.elements;
+  }
+
+  if (t.isObjectProperty(parentNode)) {
```

Comment:
To clarify, this will allow stepping to the call in `{ prop: foo() }`, but wouldn't allow `{ prop: foo() + 1 }`, is that right?

Reply:
yes... this is a bit awkward I admit :/ I guess a depth first search would be best

Reply:
I'd like to switch to a smarter system in a follow up phase. I started this branch, but got a bit tripped up

https://github.com/devtools-html/debugger.html/compare/master...jasonLaster:pp-traverse?expand=1

I'll file a follow up issue

Reply:
Sounds good!